cli/cmd/inject.go

```diff
@@ -483,3 +504,7 @@ func checkUDPPorts(t *v1.PodSpec) bool {
 	}
 	return false
 }
+
+func injectDisabled(t *metaV1.ObjectMeta) bool {
+	return t.GetAnnotations()[k8s.ProxyInjectAnnotation] == k8s.ProxyInjectDisabled
```

Comment:
Hmm, that makes sense, but I'm going to punt on it if that's ok by you, since the auto-inject implementation doesn't validate the value of this annotation either. I'd like to consider adding it as part of #1999.